utilities/bash/run_six.sh

```diff
@@ -801,14 +801,23 @@ function submitCreateFinalInputs(){
 	sixdeskmess  1 "sixdesktaskid: $sixdesktaskid - $sixdeskTaskId"
 	# - return sixdeskTaskName and workunitName
 	sixdeskDefineWorkUnitName $workspace $Runnam $sixdesktaskid
+	# check if boinc or htboinc and adjust the zip filename accordingly
```

Comment:
on this bunch of changes, I would actually allow for the case of HTBoinc to deal with the same .zip file name as BOINC. I would simply add a conditional statement for the generation of the .desc file